File: 6-Patron/php/FabricaDecorador_BendicionBotas.php
<?php
include './FabricaBotas.php';
include './Bota.php';
abstract class FabricaDecorador_BendicionBotas extends FabricaBotas{
	private $botasNormales;
	private $creaBotaAgregada;

	abstract public function crearBotas();
	abstract public function bendecirBotas();
  
}
?>

File: 6-Patron/python/FabricaDecoradorBendicionBotas.py
import FabricaBotas


class FabricaDecorador_BendicionBotas(FabricaBotas):
    def __init__(self, botasNormales, creaBotaAgregada) -> None:
        super().__init__()
        self.__botasNormales = botasNormales
        self.__creaBotaAgregada = creaBotaAgregada

    def crearBotas(self):
        pass

    def bendecirBotas(self):
        pass
